lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -135,8 +135,8 @@ export class PublicClientApplication extends ClientApplication implements IPubli
                         accessTokenSize: result.accessToken.length,
                         idTokenSize: result.idToken.length,
                         isNativeBroker: result.fromNativeBroker,
-                        cacheLookupPolicy: request.cacheLookupPolicy,
```

Comment:
Is there a reason this is being removed?

Reply:
@jo-arroyo Thanks so much for pointing this out.